### PR TITLE
[Task] Implement dark theme for cookie policy

### PR DESCRIPTION
**Pull request by ErickDoroy**
## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202364780519234/f
- Route Link: http://localhost:3000/

## Description

- Implement dark theme in the cookie policy

## Notes

N/A

## Screenshots

<details>
<summary>dark cookie policy</summary>

image
dark-cookie-policy

</details>

### CODE REVIEW

**Review by ume45876139 — APPROVED** [review]
# Working

<details>
<summary>Chrome</summary>

スクリーンショット 2022-05-31 18 52 44


</details>

<details>
<summary>Safari</summary>

スクリーンショット 2022-05-31 18 53 55

</details>

<details>
<summary>Opera</summary>

スクリーンショット 2022-05-31 18 57 45

</details>

<details>
<summary>Firefox</summary>

スクリーンショット 2022-05-31 18 56 40

</details>

- Cookie policy for dark theme color is correct
- Button is working correctly
- Also displayed at testimonial page

# Testing data
test device: Chrome
test time: 0.3
comment: 0
iteration: 0
total test time: 0.3h

File: src/pages/components/CookiePolicy.js [hunk 1]
```diff
@@ -22,18 +22,24 @@ const CookiePolicy = () => {
     <div
       className={`${
         isPopUp ? 'block' : 'hidden'
-      } fixed bottom-0 z-20 border-t-2 bg-white py-4 px-2 text-justify leading-6 md:px-12 lg:py-10 lg:px-[200px]`}
+      } fixed bottom-0 z-20 border-t-2 bg-white py-4 px-2 text-justify leading-6 dark:border-t-dark-ash-700 dark:bg-dark-ash-800 md:px-12 lg:py-10 lg:px-[200px]`}
```
Comment (egiegarciano14) [inline src/pages/components/CookiePolicy.js]:
```suggestion
      } fixed bottom-0 z-20 border-t-2 bg-white py-4 px-2 text-justify leading-6 dark:border-t-dark-ash-700 dark:bg-dark-ash-800 dark:text-white md:px-12 lg:py-10 lg:px-[200px]`}
```

File: src/pages/components/CookiePolicy.js [hunk 2]
```diff
@@ -22,18 +22,24 @@ const CookiePolicy = () => {
...
     >
-      <p className='text-sm lg:text-base'>
+      <p className='text-sm dark:text-white lg:text-base'>
```
Comment (egiegarciano14) [inline src/pages/components/CookiePolicy.js]:
```suggestion
      <p className='text-sm lg:text-base'>
```

File: src/pages/components/CookiePolicy.js [hunk 3]
```diff
@@ -22,18 +22,24 @@ const CookiePolicy = () => {
...
+          to='/privacy-policy'
+          className='text-navy-900 hover:underline dark:text-navy-100'
+          target='_blank'
+        >
           Privacy Policy
         </Link>
       </p>
       <div className='mt-6 block items-center justify-start md:flex lg:justify-between'>
-        <p className='text-sm lg:text-base'>By using our site you agree to our use of cookies.</p>
+        <p className='text-sm dark:text-white lg:text-base'>
```
Comment (egiegarciano14) [inline src/pages/components/CookiePolicy.js]:
```suggestion
        <p className='text-sm lg:text-base'>
```

**Review by egiegarciano14 — APPROVED** [review]

**Pull request merged**